app/Filament/Server/Resources/ScheduleResource.php

```diff
@@ -264,7 +264,7 @@ public static function form(Form $form): Form
                                 ->action(function (Set $set, $data) {
                                     $set('cron_minute', '0');
                                     $set('cron_hour', '0');
-                                    $set('cron_day_of_month', '0');
+                                    $set('cron_day_of_month', '*');
```

Comment:
`*` would mean every day.

```suggestion
                                    $set('cron_day_of_month', '1');
```